core/src/main/java/tc/oc/pgm/map/MapFactoryImpl.java

```diff
@@ -57,7 +57,8 @@ public class MapFactoryImpl extends ModuleGraph<MapModule, MapModuleFactory<? ex
   private FeatureDefinitionContext features;
 
   public MapFactoryImpl(Logger logger, MapSource source) {
-    super(Modules.MAP); // Do not copy to avoid N copies of the factories
+    super(
+        Modules.MAP, Modules.MAP_DEPENDENCY_ONLY); // Do not copy to avoid N copies of the factories
```

Comment:
```suggestion
    super(Modules.MAP, Modules.MAP_DEPENDENCY_ONLY); // Don't copy, avoid N factory copies
```
let's try to avoid formatter sending this to the next line